types/matches.d.ts

```diff
@@ -1,29 +1,33 @@
-export type MatcherFunction = (content: string, element: HTMLElement) => boolean;
-export type Matcher = string | RegExp | MatcherFunction;
+export type MatcherFunction = (content: string, element: HTMLElement) => boolean
+export type Matcher = string | RegExp | MatcherFunction
 
-export type NormalizerFn = (text: string) => string;
+export type NormalizerFn = (text: string) => string
 
 export interface MatcherOptions {
-    exact?: boolean;
-    /** Use normalizer with getDefaultNormalizer instead */
-    trim?: boolean;
-    /** Use normalizer with getDefaultNormalizer instead */
-    collapseWhitespace?: boolean;
-    normalizer?: NormalizerFn;
+  exact?: boolean
+  /** Use normalizer with getDefaultNormalizer instead */
+  trim?: boolean
+  /** Use normalizer with getDefaultNormalizer instead */
+  collapseWhitespace?: boolean
+  normalizer?: NormalizerFn
+  /** set query suppress suggestions */
```

Comment:
This comment doesn't make sense to me 😅

Reply:
lol sorry i rushed it you're right haha
```suggestion
  /** suppress suggestions for a specific query */
```